packages/core/package.json

```diff
@@ -44,7 +47,9 @@
     "/migrations/*.js"
   ],
   "scripts": {
-    "test": "buidler test",
+    "test": "yarn mocha-test && yarn truffle-test",
+    "truffle-test": "buidler test truffle-tests",
```

Comment:
feels like this naming is a bit strange. `truffle-test` are tests using the truffle framework, but run with buidler?

Reply:
Totally agree. The naming is weird. Funny enough, I think we _have_ to use the normal `test` directory for buidler tests. You can pass the individual test files to buidler to get it to _not_ use the test directory, but it fails to find the truffle fixture or even its own config when you do so 🤦

Reply:
Updated the scripts to be called buidler-test and mocha-test. The folders are named test (because I'm struggling to figure out how to get buidler to run tests in dirs other than `test`) and mocha-test respectively. PTAL.